### PR TITLE
gpload give wrong message if there are errors in external data format…

### PR DESCRIPTION
**Pull request by 50wu**
… w/o reuse_table.

If there is no reuse_table, there would be no way to check the format error. If resue_table is true,
gpload can show how to check the detail error message.

Gpload can't print format error message if the external table is deleted (without reuse_table). This commit fix the message printed by goload.

### CODE REVIEW

**Comment by adam8157** [comment]
LGTM.

Does anyone have a better idea to handle the error logs of dropped temp table?

File: gpMgmt/bin/gpload.py [hunk 1]
```diff
@@ -2332,25 +2333,35 @@ def create_staging_table(self):
...
-            if self.log_errors:
-                self.log(self.WARN, "please use GPDB built-in function gp_read_error_log('%s') to access the detailed error row" % self.extTableName)
-            self.exitValue = 1
         elif errors:
             self.log(self.WARN, '%d bad rows'%errors)
-            if self.log_errors:
-                self.log(self.WARN, "please use GPDB built-in function gp_read_error_log('%s') to access the detailed error row" % self.extTableName)
-            self.exitValue = 1
+
+        # error message is also deleted if externatal table is dropped.
```
Comment (danielgustafsson) [inline gpMgmt/bin/gpload.py]: s/externatal/external/

**Comment by ascherbaum-pivotal** [comment]
Except the comment from Daniel: looks good

**Comment by 50wu** [comment]
Merged into master

**Pull request closed without merging**